frontend/src/metabase/query_builder/components/view/ViewHeader/components/ViewTitleHeaderRightSide/ViewTitleHeaderRightSide.tsx

```diff
@@ -178,15 +180,16 @@ export function ViewTitleHeaderRightSide({
           onCloseSummary={onCloseSummary}
         />
       )}
-      {QuestionNotebookButton.shouldRender({
-        question,
-        isActionListVisible,
-      }) && (
-        <QuestionNotebookButton
-          isShowingNotebook={isShowingNotebook}
-          setQueryBuilderMode={setQueryBuilderMode}
-        />
-      )}
+      {!isBrandNew &&
+        QuestionNotebookButton.shouldRender({
```

Comment:
`isBrandNew` check should be a part of `QuestionNotebookButton.shouldRender`.

We should also make sure there is a test for `QuestionNotebookButton` being shown in interactive embedding context (see https://github.com/metabase/metabase/blob/49926e687366b0c67eb6748b4b154a32933167bf/enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/NotebookButton.tsx).

Reply:
I originally had the check in `QuestionNotebookButton.shouldRender`, but then I realized it would require passing `result` and `queryBuilderMode` into `NotebookButton`, and those variables are not necessarily relevant to an embedded notebook button. Conceptually, if you're using the notebook button in embedding, you actually might not care if something is brand new, right? That's why I kept it only in our query builder. 

I'll ask @metabase/embedding  team what they think.

Reply:
> it would require passing `result` and `queryBuilderMode` into `NotebookButton`, and those variables are not necessarily relevant to an embedded notebook button.

We only need to pass `isBrandNew`. We can hardcode it in embedding context, similar to `isActionListVisible`.

Reply:
> We should also make sure there is a test for `QuestionNotebookButton` being shown in interactive embedding context 

:pray:

Reply:
@kamilmielnik I don't understand what there is to test about this behavior being different in interactive embedding? Do you mean adding tests for SDK components that should display `NotebookButton` ?